File: official(c)/Junior/j2.cpp
#include <iostream>
using namespace std;

int main() {
    int d, e;
    cin >> d >> e;
    int current = d;
    
    for (int i = 0; i < e; i++) {
        string op;
        int q;
        cin >> op >> q;
        if (op == "+") {
            current += q;
        } else {
            current -= q;
        }
    }
    
    cout << current << endl;
    return 0;
}

File: official(c)/Junior/j2.java
import java.util.Scanner;
public class j2 {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int d = input.nextInt();
        int e = input.nextInt();
        int current = d;
        
        for (int i = 0; i < e; i++) {
            String op = input.next();
            int q = input.nextInt();
            if (op.equals("+")) {
                current += q;
            } else {
                current -= q;
            }
        }
        
        System.out.println(current);
        input.close();
    }
}